subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -34,6 +34,53 @@ config BT_ATT_TX_MAX
 	  amount the calls will block until an existing queued PDU gets
 	  sent.
 
+config BT_EATT
+	bool "Enhanced ATT Bearers support"
+	default y
```

Comment:
Well I guess we want to promote the use of EATT since it does have benefits over regular ATT and the footprint is really minimal when BT_L2CAP_DYNAMIC_CHANNEL is already enabled, it is kind weird to see this sort of comment comming from you btw, since default n tends to create a lot more problem down the road, for instance if I had done this since the beginning EDTT sample would be broken given it wouldn't even test EATT.

Reply:
I think any feature addition should be default to `n` and then we can enable the feature for the tests.

Reply:
Are you saying that because it is experimental it should default to n? There are already testing for it since all tests running GATT tests will essentially enable it, anyway it is already gated by BT_L2CAP_DYNAMIC_CHANNEL, that said maybe we can wait to enable it when qualification is done but that usually is a past a release so that means no new feature can default to y until qualified.

Reply:
I marked it as EXPERIMENTAL like it was done for Advertising Extensions.